lane.py

```diff
@@ -1869,6 +1869,8 @@ def _modify_loading(cls, qu):
             joinedload(license_pool_name, "delivery_mechanisms"),
             joinedload(license_pool_name, "delivery_mechanisms", "delivery_mechanism"),
 
+            joinedload(license_pool_name, "identifier"),
```

Comment:
This lets us access active_license_pool.identifier from OPDS generation code without making a separate database query -- the data is preloaded as part of the original lookup.